components/feature/tabs/src/main/java/mozilla/components/feature/tabs/TabsUseCases.kt

```diff
@@ -80,6 +80,16 @@ class TabsUseCases(
          */
         operator fun invoke(sessionId: String)
 
+        /**
+         * Removes the session with the provided ID. This method
+         * has no effect if the session doesn't exist.
+         *
+         * @param sessionId The ID of the session to remove.
+         * @param selectParentIfExists Whether or not to select the parent tab
+         * of the removed tab if a parent exists.
+         */
+        operator fun invoke(sessionId: String, selectParentIfExists: Boolean) = invoke(sessionId)
```

Comment:
I am only adding this for the method accepting a `sessionId` as we want to remove the one accepting a browser session anyway. I also provide a default implementation to prevent a breaking API change for `TabsTrayFragment` and `RemoveTabWithUndoUseCase` which also shouldn't have to implement this method.

Reply:
Do we care that the default implementation will ignore `selectParentIfExists`? 🤔

Reply:
Ideally, yeah, but the only other option is to break API now and then later again. Unfortunately, this type was sub-classed in both A-C and Fenix so I was trying to avoid that... 🤔

Reply:
OK, I documented this as a temporary workaround, and that we should remove the default implementation when we remove invoke(session: Session).